### PR TITLE
pythonPackages.pomegranate: init at 0.7.7

### PR DESCRIPTION
**Pull request by rybern**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/pomegranate/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{ stdenv, pythonPackages, fetchFromGitHub }:
```
Comment (FRidh) [inline pkgs/development/python-modules/pomegranate/default.nix]: Separate parameters are required, not a single package set.

File: pkgs/development/python-modules/pomegranate/default.nix [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  src = fetchFromGitHub {
+    repo = "pomegranate";
+    owner = "jmschrei";
+    rev = version;
+    sha256 = "07zkg9jxjql00ppmkc4wsm2xidx58xnkmbbqaqv8jmh67y80yx75";
+  };
+
+  propagatedBuildInputs = with pythonPackages; [ numpy scipy cython networkx joblib ];
+
+  buildInputs = [ pythonPackages.nose ];
```
Comment (FRidh) [inline pkgs/development/python-modules/pomegranate/default.nix]: checkInputs

File: pkgs/development/python-modules/pomegranate/default.nix [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{ stdenv, pythonPackages, fetchFromGitHub }:
+
+pythonPackages.buildPythonPackage rec {
+  pname = "pomegranate";
+  version = "0.7.7";
+  name  = "${pname}-${version}";
+
+  src = fetchFromGitHub {
```
Comment (FRidh) [inline pkgs/development/python-modules/pomegranate/default.nix]: any reason to fetch from GitHub instead of PyPI?

**Review by FRidh — CHANGES_REQUESTED** [review]

**Comment by rybern** [comment]
@FRidh fixed I think

**Comment by rybern** [comment]
@FRidh Anything else to change?

**Pull request merged**